ci-operator/step-registry/openshift/e2e/aws/builds/techpreview/openshift-e2e-aws-builds-techpreview-workflow.yaml

```
@@ -0,0 +1,14 @@
+workflow:
+  as: openshift-e2e-aws-builds-techpreview
+  steps:
+    pre:
+    - chain: ipi-aws-pre-techpreview
+    test:
+    - ref: openshift-e2e-test
+    post:
+    - chain: gather-core-dump
+    - chain: ipi-aws-post
```

Why do we need to gather the core dump? `ipi-aws-post` gives us the must-gather archive, which is sufficient for our testing needs.
```suggestion
    - chain: ipi-aws-post
```

---

it is not needed indeed